doc: prepare new release 3.8.2 (#4683)
tracim/tracim: changelog.md

# 3.8.2 / 2021-05-27

### New Features

> - Add link to the public profile via favorites page
> - Configuration parameter (in server config) permit to disable the space parent input when creating a space

Reviewer: ```suggestion
- Add links to the public profile in the favorites page
- Configuration parameter (in server config) allows disabling the space parent input when creating a space
```

Author: Done

### Fixed Issues

> UX:

Reviewer: If this does not gets filled, don't forget to remove this

### Breaking/Important change

> - All necessary files to customize Tracim instance (logo, color) are now available in `branding/` folder. Some manually change is need to run this version (same process for local installation or docker installation):

Reviewer: ```suggestion
- All necessary files to customize a Tracim instance (logo, color) must be placed in the `branding/` folder. A `branding.sample` folder is given as an example. Some manual changes are required to run this version (same process for local installation or docker installation):
```

Author: Done.

  - for Tracim >= 3.8.0

>   :warning: if you not apply that, Tracim cant start and use your customization

Reviewer 2: ```suggestion
  :warning: if you not apply that, Tracim won't start and use your customization
```

Author: Done

>     - First, it's necessary to backup the existing branding folder if you have added some customization (if not, its necessary to delete branding folder `rm -r /{docker-volume}/etc/branding`). To do that, you need to:

Reviewer: ```suggestion
    - First, it's necessary to backup the existing branding folder if you have added some customization (if not, it's necessary to delete the branding folder: `rm -r /{docker-volume}/etc/branding`). To do that, you need to:
```

Author: Done

      - `mv /{docker-volume}/etc/branding/ /{docker-volume}/etc/branding-back/`

>       - The `branding/` folder is created automatically by Tracim if it not exists

Reviewer: ```suggestion
      - The `branding/` folder is created automatically by Tracim if it does not exist
```

    - Second, start Tracim 3.8.2 and after that you need to:
      - if you have customized your `logo.png`:
        - `mv /{docker-volume}/etc/logo.png /{docker-volume}/etc/branding/images/tracim-logo.png`
      - if you have customized your `color.json`:
        - its better to change the new `color.json` file available in `/{docker-volume}/etc/branding/` (the new key `"sidebar":` is mandatory and available by default since Tracim 3.8.1)
      - if you have customized some files in the old `branding/` folder, you need to copy your file from `/{docker-volume}/etc/branding-back/` folder
    - Finally, restart Tracim
  - for Tracim < 3.8.0, if you have customized `logo.png` and/or `color.json`

>   :warning: if you not apply that, Tracim cant use your customization

Reviewer 2: ```suggestion
  :warning: if you not apply that, Tracim can't use your customization
```

Author: done

    - After first start of the version 3.8.2, you need to:
      - if you have customized your `logo.png`:
        - `mv /{docker-volume}/etc/logo.png /{docker-volume}/etc/branding/images/tracim-logo.png`
      - if you have customized your `color.json`:

>         - its better to change the new `color.json` file available in `/{docker-volume}/etc/branding/` (the new key `"sidebar":` is mandatory and available by default since Tracim 3.8.1)

Reviewer: ```suggestion
        - it's better to change the new `color.json` file available in `/{docker-volume}/etc/branding/` (the new key `"sidebar":` is mandatory and available by default since Tracim 3.8.1)
```

Author: Done

- Apache: default configuration change. You need to delete your config file `/{docker-volume}/etc/apache2.conf` (backup your file first) before starting/restarting the docker image

> :warning: if you not apply this, Tracim cant show the `favicon.ico`

Reviewer 2: ```suggestion
:warning: if you not apply this, Tracim can't show the `favicon.ico`
```

Author: done
